README.md

```diff
@@ -10,16 +10,25 @@ Follow the steps below to get started with the Node Template, or get it up and r
 your browser in just a few clicks using
 the [Substrate Playground](https://docs.substrate.io/playground/) :hammer_and_wrench:
 
-### Using Nix
+### Repository Set-Up
 
-Install [nix](https://nixos.org/) and optionally [direnv](https://github.com/direnv/direnv) and
-[lorri](https://github.com/target/lorri) for a fully plug and play experience for setting up the
-development environment. To get all the correct dependencies activate direnv `direnv allow` and
-lorri `lorri shell`.
+After cloning this repository, be sure to include necessary submodules.
+
+```sh
+git submodule init
+git submodule update
+```
```

Comment:
this is really good. we may want to put these commands inside a generic `setup.sh` and have the readme point to running that -- this way, if we ever need to remove or add functions for getting the repo "setup", we just edit setup.sh, instead of having to add/remove steps from the README.md

Reply:
Yeah we have a `build.sh` script which should do parts of the setup/build process - changing the name and adding the submodule setup is a great idea though.

Reply:
If we have a setup.sh, we could probably then possibly reduce the "setup" section down to "run setup.sh" and include comments in the script in case people want to know specifically what the script is doing. Also, do the same for build.sh, and test.sh. And for a deeper dive for developers, such as how to contribute or whatever we may think up, maybe that goes into the wiki?

Reply:
What problem does this solve? Why is this valuable? Why should you care? What should you do about it?

Can we answer that in 3-5 sentences?

Reply:
Good call. That makes sense, we have a scripts directory anyway that's not used since we overwrote the original template scripts.